test/SlowTests/Issues/RavenDB-17597.cs

```diff
@@ -258,7 +303,7 @@ public async Task ModifyIndexThenRestartServer(bool stopIndex)
                         store.Maintenance.Send(new StopIndexOperation("Items/ByNum"));
                     }
                 }
-                catch (Exception e)
+                catch (Exception)
```

Comment:
remove the `try/catch` here, we don't expect an error here.

Reply:
Done